src/dictionaries.py

```diff
@@ -36,9 +36,18 @@
 # Add a new waypoint to the list
 # YOUR CODE HERE
 
+waypoints.insert(0, {'lat': 0, 'lon': 0, 'name': 'orgin'})
+
 # Modify the dictionary with name "a place" such that its longitude
 # value is -130 and change its name to "not a real place"
 # YOUR CODE HERE
 
+for i in waypoints:
+    if (i['name'] == 'a place'):
+        i['name'] = 'not a real place'
+        i['lon'] = '-130'
+
 # Write a loop that prints out all the field values for all the waypoints
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+for i in waypoints:
+    print(i.values())
```

Comment:
You could also do this which formats a bit differently `for w in waypoints:
    print(w['name'], w['lat'], w['lon'])`